packages/main/src/TabContainer.hbs

```diff
@@ -45,7 +45,7 @@
 			</ul>
 		</div>
 
-		<ui5-icon @click="{{_onHeaderForwardArrowClick}}" class="{{classes.headerForwardArrow}}" src="sap-icon://slim-arrow-right" tabindex="-1"></ui5-icon>
+		<ui5-icon @click="{{_onHeaderForwardArrowClick}}" class="{{classes.headerForwardArrow}}" src="sap-icon://slim-arrow-right" tabindex="-1" accessible-name="Next" icon-tooltip="true"></ui5-icon>
```

Comment:
Actually we changed the icon-tooltip to show-tooltip. You see the documentation of the Icon component